src/api/types.ts

```diff
@@ -36,15 +36,18 @@ export interface UserRecord {
 
 export interface StatusRecordInput {
   title: string;
+  color: string;
 }
 
 export interface StatusRecordOutput {
   id: number;
   object_uuid: string;
   title: string;
+  color: string;
   created: string;
   activated?: string;
   deactivated?: string;
+  due_date?: string;
```

Comment:
Not sure why a status should have a due_date.

Reply:
Whoops, should be for WriteTicketRecord.